bittensor/_cli/cli_impl.py

```diff
@@ -239,7 +239,7 @@ def register( self ):
         """
         wallet = bittensor.wallet( config = self.config )
         subtensor = bittensor.subtensor( config = self.config )
-        subtensor.register( wallet = wallet, prompt = not self.config.no_prompt, num_processes = self.config.num_processes, update_interval = self.config.update_interval )
+        subtensor.register( wallet = wallet, prompt = not self.config.no_prompt, num_processes = self.config.get('num_processes', None), update_interval = self.config.get('update_interval', None) )
```

Comment:
shouldnt we set a default for num_process and update_interval at the config so that we dont have to handle none?
cause as i rmb, the num_process and update interval will be overrided if they are none withing the registration function anyways

Reply:
yes, they override at the registration function. We can add a flag to the run command as well, but I figured it best to address the fix in this PR and then add the flag